src/statick_tool/plugins/tool/clang_format.py

```diff
@@ -184,14 +184,14 @@ def check_configuration(self, clang_format_bin: str) -> Optional[bool]:
         if self.plugin_context is None:
             return False
 
+        default_file_name = "_clang-format"
+        format_file_name = self.plugin_context.resources.get_file(default_file_name)
+        exc_msg = (
+            "_clang-format or .clang-format style is not correct. "
+            f"There is one located in {format_file_name}. "
+            "Put this file in your home directory."
+        )
         try:
-            default_file_name = "_clang-format"
-            format_file_name = self.plugin_context.resources.get_file(default_file_name)
-            exc_msg = (
-                "_clang-format or .clang-format style is not correct. "
-                f"There is one located in {format_file_name}. "
-                "Put this file in your home directory."
-            )
 
             if not os.path.isfile(os.path.expanduser("~/" + default_file_name)):
```

Comment:
```suggestion
            if not os.path.isfile(os.path.expanduser("~/" + default_file_name)):
```
nit: I don't think this blank line is needed.

Reply:
[eac261b](https://github.com/triboelectric/statick/pull/540/commits/eac261b581a1c36d0abfef0ca0f49445b18ded0d)